api/pacman-api-admin/src/main/java/com/tmobile/pacman/api/admin/domain/CreateAccountRequest.java

```diff
@@ -29,5 +29,9 @@ public class CreateAccountRequest {
     private String tenableAccessKey;
     private String tenableSecretKey;
 
+    private String redhatAccountId;
+    private String redhatAccountName;
+    private String redhatToken;
+
```

Comment:
there should be another **optional** field "owner" that the API can expect.

Reply:
But in the last call, did we decide to keep it out, right? Because even when we accept the owner from the UI, we don't have a field in DB to save the value.